### PR TITLE
[sub tatsk] Add library files to the project

### PR DESCRIPTION
**Pull request by chibujax**
# JIRA TICKET NAME:
PDMO-10: [Add library files to the project and clean up merge conflict comments on files](https://issues.openmrs.org/browse/PDMO-10)

## SUMMARY:
Add library files to the project and clean up merge conflict comments on files

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```
Comment (dkayiwa) [inline webpack.config.js]: This license version does not look correct. Did you confirm that this is the latest license that we are using?

Comment (chibujax) [reply webpack.config.js]: I have updated to the version. The license were auto-generated, but I have updated them now.

Comment (dkayiwa) [reply webpack.config.js]: auto generated by what?

Comment (chibujax) [reply webpack.config.js]: https://www.npmjs.com/package/generator-openmrs-owa . We used OpenMRS Open Web App Generator

Comment (dkayiwa) [reply webpack.config.js]: Can you create a new talk post and mention about the need to have this fixed to use the latest license?

Comment (chibujax) [reply webpack.config.js]: ok I will. Please any luck with the two PRs?  😄

Comment (dkayiwa) [reply webpack.config.js]: First do that talk post.

Comment (chibujax) [reply webpack.config.js]: I have. https://talk.openmrs.org/t/openmrs-open-web-app-generator-with-outdated-license/12886

File: README.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 Open Web Application for managing patient observations data
-=======
 <img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
 
 # Obsadmin
 
 This repository contains the Obsadmin OpenMRS Open Web App.
 
-> Add a description of what your app does here.
+Add a description of what your app does here.
```
Comment (dkayiwa) [inline README.md]: Weren't you supposed to replace the above line?

Comment (chibujax) [reply README.md]: Updated

**Pull request merged**